pkg/features/antrea_features.go

```diff
@@ -136,6 +136,14 @@ var (
 		SecondaryNetwork:  {},
 		ServiceExternalIP: {},
 	}
+	// supportedFeaturesOnExternalNode records the features supported on a VM or a BM
+	// host. Antrea Agent checks the enabled features if it is running as VM
+	// or BM agent role, and fails the startup if an unsupported feature is
+	// enabled.
+	supportedFeaturesOnExternalNode = map[featuregate.Feature]struct{}{
```

Comment:
Again, let us use one term - VirtualMachine, or VMNode if you prefer.

Reply:
updated.